react-components/src/components/CameraController/CameraController.tsx

```diff
@@ -34,7 +51,7 @@ export function CameraController({ initialFitCamera }: CameraControllerProps): R
       return;
     }
     if (fittingStrategy.to === 'allModels' && modelsAdded) {
-      viewer.fitCameraToModels(viewer.models, 0, true);
+      viewer.fitCameraToModels(undefined, undefined, true);
```

Comment:
doesn't seems right passing `undefined` as a parameter even though `fitCameraToModels` validate internally and assigns default values

Reply:
It fixes a bug if you pass models list. We don't have other option beside passing undefined since we want to have last argument defined. I saw that as normal practice outside our codebase.

Reply:
The bug is that it is not taking into account last added model.

Reply:
If bug is related to last added model is missing then how would passing `undefined` fix it, `viewer.models` will contain all the models added into the `Cognite3DViewer`

Reply:
Not sure why it was the case, you can test it yourself. I decided not to spend too much time on that and changed the logic accordingly.

Reply:
I still not convinced, but to move forward I am ok with it 👍